misc/formatter/modules/formatter-core/src/test/resources/testng.xml

```diff
@@ -29,6 +29,7 @@ under the License.
         <packages>
             <package name="org.ballerinalang.formatter.core.types.*" />
             <package name="org.ballerinalang.formatter.core.expressions.*" />
+            <package name="org.ballerinalang.formatter.core.statements.*" />
```

Comment:
We might have to leave this out until fixes for the rest of the statement tests are merged, if there are failing statement tests.

Reply:
Fixed by limiting to `org.ballerinalang.formatter.core.statements.XMLNSDeclarationStatementsTest` only.

Reply:
Removed the `statement` package and added the `XMLNSDeclarationStatementsTest` class separately.